docs/source/development/traffic_portal.rst

```diff
@@ -104,13 +104,13 @@ Installing The Traffic Portal Developer Environment
 #. Make sure that compass is installed and functioning correctly by running ``compass version``. If compass is not available, then it can be installed following the instructions under :ref:`dev-tp-compass`.
 
 #. Run ``grunt`` to package the application into ``traffic_portal/app/dist``, start a local HTTPS server (Express), and start a file watcher.
```

Comment:
can you take a look at these steps. they don't seem quite right. 

![image](https://user-images.githubusercontent.com/251272/103377114-b3b06400-4a9b-11eb-9a2f-5a549774237b.png)

i.e. 6 should be after 7 and the subbullets are screwed up

Reply:
otherwise, all looks good.